convokit/coordination/coordination.py

```diff
@@ -101,16 +101,29 @@ def fit_transform(self, corpus: Corpus, y=None) -> Corpus:
         self.fit(corpus)
         return self.transform(corpus)
 
-    def score(self, corpus: Corpus, speakers: Collection[Union[Speaker, str]],
+    def summarize(self, corpus: Corpus, speakers: Collection[Union[Speaker, str]],
               group: Collection[Union[Speaker, str]], focus: str = "speakers",
-              speaker_thresh: int = 0, target_thresh: int = 3,
-              utterances_thresh: int = 0, speaker_thresh_indiv: int = 0,
-              target_thresh_indiv: int = 0,
-              utterances_thresh_indiv: int = 0,
+              summary_report: bool = False,
+              speaker_thresh: int = -1, target_thresh: int = -1,
+              utterances_thresh: int = -1, speaker_thresh_indiv: int = -1,
+              target_thresh_indiv: int = -1,
+              utterances_thresh_indiv: int = -1,
```

Comment:
Could we use the Optional[int] style here?

Reply:
updated!

Reply:
Ah, but this also means the default should then be None instead of -1.

Reply:
fixed